app/scripts/modules/amazon/src/pipeline/stages/resizeAsg/resizeAsgStepLabel.html

```diff
@@ -1,5 +1,8 @@
-<span class="task-label">
+<span class="task-label" ng-if="!step.context.useNameAsLabel">
   Resize Server Group:
   {{step.context.serverGroupName}}
   ({{step.context.region}})
 </span>
+<span class="task-label" ng-if="step.context.useNameAsLabel">
```

Comment:
I don't really like having to send an attribute back from `orca` indicating how I want a UI label to render, but I also don't really like that it disregards the specified stage name for this particular stage.

![image](https://user-images.githubusercontent.com/388652/32472137-e20c211a-c315-11e7-8a52-331e92918fc9.png)

In RRB, we'll get `Grow to X% of Desired Size` rather than `Resize Server Group: ...`.

Thoughts?

Reply:
I don't love doing it, either, but we do similar things with Manual Judgment stages and some other places I can't think of right now, so it's fine in my opinion.

Reply:
If such a thing comes up in the future (or we ever wish to retrofit this), I believe we should be able to accomplish something similar in `orca` using expressions to grab the server group name once it appears in the context.

Ideally such business doesn't not belong in `deck`.